ChatQnA/benchmark_results.md

```diff
@@ -0,0 +1,161 @@
+| **Containers and Virtualization** | Kubernetes v1.29.9                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| **Drivers**                       | habana driver 1.20.1-366eb9c                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| **VM vcpu, Memory**               | 160 vCPUs, 1T memory                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| **OPEA Release Version**          | v1.3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| **Dataset**                       | pubmed_10.txt                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
As reranking model is not mentioned, does this mean that no dataset was prepared for ChatQnA app, and therefore it did not use reranking?